gufe/components/smallmoleculecomponent.py

```diff
@@ -247,6 +261,7 @@ def _from_dict(cls, d: dict):
             a.SetChiralTag(_INT_TO_ATOMCHIRAL[atom[4]])
             a.SetAtomMapNum(atom[5])
             _setprops(a, atom[6])
+            a.SetHybridization(_INT_TO_HYBRIDIZATION[atom[7]])
```

Comment:
We should put this in a try/except with a warning.

Reply:
Yeah this is why the tests are currently failing as the old graphml files don't have the hybridization info, I think try-except is the best option so I'll add that in.

Reply:
Thanks!

Reply:
done in [27bd913](https://github.com/OpenFreeEnergy/gufe/pull/408/commits/27bd913c3eaa01a4ee5183ffb9265d7a22978a24)